web/src/components/Article/components/ArticlePhotoGallery.tsx

```diff
@@ -0,0 +1,120 @@
+import ArticleTypeIcon, {
+  EPostType,
+} from 'src/components/ArticleTypeIcon/ArticleTypeIcon'
+import Avatar from 'src/components/Avatar/Avatar'
+import Button from 'src/components/Button/Button'
+import PhotoGrid from 'src/components/PhotoGrid/PhotoGrid'
+import { EPostDisplayType } from 'src/types/post-display-type.enum'
+
+interface Props {
+  article: Post
+  displayType: EPostDisplayType
+  date?: string
+}
+
+const ArticlePhotoGallery = ({ article, displayType, date }: Props) => {
+  const authorName =
+    article.user?.profile?.name || article.user?.name || 'Anonymous'
+
+  const previewGallery = article?.photoGallery?.slice(0, 5)
+
+  return (
+    <>
+      {displayType === EPostDisplayType.PREVIEW && (
+        <>
+          <div className="relative ">
+            <PhotoGrid
+              photoGallery={previewGallery}
+              preview={true}
+              className="block h-full w-full"
+            />
+            <div className="font-3xl absolute bottom-0 z-10 mx-auto h-full w-full max-w-screen-xl bg-gray-600 bg-opacity-50 px-4 py-20 text-center text-white text-opacity-100 md:py-24 lg:py-56">
+              <div className="flex flex-row items-center justify-center gap-2 pb-2">
+                <div>
+                  <ArticleTypeIcon type={article.type as EPostType} />
+                </div>
+
+                <h1 className="flex flex-wrap items-center justify-center text-3xl font-extrabold leading-none tracking-tight text-white md:text-5xl lg:text-6xl">
+                  {article.title}
+                </h1>
+              </div>
+              <div className="flex flex-row items-center justify-center gap-12">
+                <div className="flex flex-row items-center gap-2">
+                  <Avatar
+                    src={article.user?.profile?.avatar}
+                    alt={article.user.name}
+                    name={article.user.name || article.user.email}
+                  />
+                  <div className="flex flex-col items-start">
+                    <span className="text-sm text-slate-300" title={authorName}>
+                      {authorName}
+                    </span>
+                    <span
+                      className="text-sm text-slate-300"
+                      title={new Date(article.createdAt).toLocaleString(
+                        'nl-NL',
+                        {
+                          hour: '2-digit',
+                          minute: '2-digit',
+                        }
+                      )}
+                    >
+                      {' '}
+                      {new Date(article.createdAt).toLocaleDateString(
+                        'nl-NL'
+                      )}{' '}
+                      {new Date(article.createdAt).toLocaleTimeString('nl-NL', {
+                        hour: '2-digit',
+                        minute: '2-digit',
+                      })}
```

Comment:
We are using these date formatters now so often that we might want to think about creating helper functions in `web/src/lib/formatters.tsx`. 

Additionally, we might also want to think about using the `ago` pattern, e.g. `5 min ago`, `5 days ago`, `3 weeks ago`, `3 months ago`, `a year ago`, etc. :D See this for example: https://stackoverflow.com/questions/3177836/how-to-format-time-since-xxx-e-g-4-minutes-ago-similar-to-stack-exchange-site